src/datepicker/datepicker.ts

```diff
@@ -94,7 +94,7 @@ export interface NgbDatepickerNavigateEvent {
       <ng-template ngFor let-month [ngForOf]="months" let-i="index">
         <div class="ngb-dp-month d-block ml-3">            
           <div *ngIf="navigation !== 'select' || displayMonths > 1" class="ngb-dp-month-name text-center">
-            {{ i18n.getMonthFullName(month.number) }} {{ month.year }}
+            {{ i18n.getMonthFullName(month.number) }} <span class="ngb-dp-year">{{ month.year }}</span>
```

Comment:
If we go this route, wouldn't it be worthwhile to also have a span element around the month name with an appropriate class too?